### PR TITLE
Milestone 10: Responsive Menu

### PR DESCRIPTION
**Pull request by amarmahdi1112**
## Responsive Menu

In this milestone:
- menu made responsive with css and js

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @amarmahdi1112 and @RuriEnciso, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have `ESlint` setup in your `linters.yml`. Kindly make sure to set it up as it is designed to assist in writing code that aligns with industry standards and best practices.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -13,13 +13,17 @@
...
-        <nav class="nav">
-          <ul>
+        <nav class="nav nav-hide" id="show-nav">
+          <div class="nav-close" id="close">
+            <img src="./app/assets/pngs/icons/close.png" alt="" srcset="">
+          </div>
+          <ul class="menu">
             <li><a href="#">Portfolio</a></li>
             <li><a href="#">About</a></li>
             <li><a href="#">Contact</a></li>
```
Comment (levy002) [inline index.html:29]: - [ ] Kindly make sure when any navigation link is clicked the menu is closed automatically and the user will be redirected to the part of the page according to the clicked nav link, if the `About` link is clicked the menu will be closed and the user must be redirected to the `About` section on your portfolio.

   ### YOUR WEBSITE BEHAVIOUR

    chrome-capture-2023-4-23 (2)

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @amarmahdi1112   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**